api/handler/v1beta1/approval.go

```diff
@@ -77,7 +77,7 @@ func (s *GRPCServer) UpdateApproval(ctx context.Context, req *guardianv1beta1.Up
 			return nil, status.Errorf(codes.InvalidArgument, "unable to process the request: %v", err)
 		case appeal.ErrActionForbidden:
 			return nil, status.Error(codes.PermissionDenied, "permission denied")
-		case appeal.ErrApprovalNameNotFound:
+		case appeal.ErrApprovalNotFound:
 			return nil, status.Errorf(codes.NotFound, "appeal not found: %v", id)
```

Comment:
Is this correct?  it is checking for `ErrApprovalNotFound`, but returning back `"appeal not found"`